pkg/mountutil/mountutil_freebsd.go

```diff
@@ -66,3 +74,65 @@ func ProcessFlagTmpfs(s string) (*Processed, error) {
 func ProcessFlagMount(s string, volStore volumestore.VolumeStore) (*Processed, error) {
 	return nil, errdefs.ErrNotImplemented
 }
+func ProcessSplit(s string, volStore volumestore.VolumeStore) (Processed, error) {
```

Comment:
I think the `ProcessSpllit` code is as same as the code in `mountutil_linux.go`. FYI https://github.com/containerd/nerdctl/pull/924/files#diff-ab0f47dd8154315baddf1347290f120ed5e9975a660c4bfb838d17dcd2418c50R434

Maybe we can add a new file named `mountutils_otthers.go` and add `//go:build !windows` at the top of the code and  move `ProcessSpllit` here

Reply:
Or maybe we can keep the conditions of ProcessSplit in the mountutil.go and keep the windows one as it is.

Reply:
Yes, I think it's a better choice